pennylane/ops/qutrit/observables.py

```diff
@@ -14,13 +14,26 @@
 """
 This submodule contains the qutrit quantum observables.
 """
-from pennylane import numpy as np
+import numpy as np
 
 import pennylane as qml  # pylint: disable=unused-import
+from pennylane.operation import Observable
 from pennylane.ops.qubit import Hermitian
 from pennylane.ops.qutrit import QutritUnitary
 
 
+# Array containing all 8 Gell-Mann matrices
+gm_mats = np.zeros((8, 3, 3), dtype=np.complex128)
+gm_mats[0] = np.array([[0, 1, 0], [1, 0, 0], [0, 0, 0]])
+gm_mats[1] = np.array([[0, -1j, 0], [1j, 0, 0], [0, 0, 0]])
+gm_mats[2] = np.diag([1, -1, 0])
+gm_mats[3] = np.array([[0, 0, 1], [0, 0, 0], [1, 0, 0]])
+gm_mats[4] = np.array([[0, 0, -1j], [0, 0, 0], [1j, 0, 0]])
+gm_mats[5] = np.array([[0, 0, 0], [0, 0, 1], [0, 1, 0]])
+gm_mats[6] = np.array([[0, 0, 0], [0, 0, -1j], [0, 1j, 0]])
+gm_mats[7] = np.diag([1, 1, -2]) / np.sqrt(3)
```

Comment:
Is this some well known order in literature? Or was it assigned from some formula? Maybe worth having the formula which takes index --> GM matrix in the docs? Although we have them explicitly listed so its not a major concern

Reply:
It seems to be a fairly common ordering; I'm actually not sure if it's derived from anything. But yeah it wouldn't hurt to put a comment both here for developers, and then in the actual docstring as well.

Reply:
The ordering follows the same comvention as the Pauli matrices. I'll add this as a comment to the file as well, but essentially, matrices 0, 3, and 5 are qutrit analogs of Pauli X with each matrix spanning a different subspace, matrices 1, 4, and 6 are qutrit analogs of Pauli Y, and matrices 2 and 7 are analogs of Pauli Z. There are only 2 matrices to generalize Pauli Z because only 8 matrices are needed for 3 dimensions to create a complete basis.